daprdocs/content/en/operations/components/certification-lifecycle.md

```diff
@@ -47,6 +47,7 @@ All components start at the Alpha stage.
 - The component must have component [certification tests](#certification-tests) validating functionality and resiliency
 - The component is maintained by Dapr maintainers and supported by the community
 - The component is well documented and tested
+- The component has been available as Alpha or Beta for at least 1 minor version release of Dapr runtime prior
```

Comment:
Please see https://github.com/dapr/dapr/issues/4970. Once that issue is resolved we should document the requirements listed there.

Reply:
Agreed.